package.json

```diff
@@ -19,9 +19,12 @@
     "lint": "tslint -p tsconfig.json"
   },
   "dependencies": {
-    "noble-ed25519": "^1.0.2"
+    "noble-ed25519": "^1.0.2",
+    "raw-body": "^2.4.1"
```

Comment:
Is `raw-body` used anywhere?

Reply:
No, `raw-body` was used by me when trying to make everything work without using a body parser to keep everything as low-level as possible but that ultimately didn't work and then I seemed to have forgot to remove `raw-body` from the dependencies.